### PR TITLE
Fixes for #42 & #43

### PR DESCRIPTION
**Pull request by mirceaalexandru**
- Seneca web path parameters should use same syntax for Hapi/Express paths #42
- Add query, body payload or path parameters to seneca action arguments in similar way as for Express #43

### CODE REVIEW

**Comment by mcdonnelldean** [comment]
LGTM

File: test/configure-server.js [hunk 1]
```diff
@@ -48,6 +48,18 @@ exports.init = function(done){
         done(null,{d0:args.d0+'f0'})
       })
 
+      this.add('role:api,cmd:x1',function(args,done){
+        done(null,{x: args.x})
```
Comment (geek) [inline test/configure-server.js]: We should remove the `test` folder to `.eslintignore`  

Please add spaces after `,` and between `function` and `(`

Comment (geek) [inline test/configure-server.js]: use a different name other than `done` for the callback, its harder to read with the test `done` function

File: web.js [hunk 2]
```diff
@@ -441,6 +455,17 @@ module.exports = function (options) {
         done()
       })
     }
+
+    function changeToHapi (path) {
+      var tokens = path.split('/')
+
+      for (var i in tokens) {
+        if (tokens[i].indexOf(':') === 0) {
```
Comment (geek) [inline web.js]: does the `fullurl` contain the protocol?

File: web.js [hunk 3]
```diff
@@ -441,6 +455,17 @@ module.exports = function (options) {
         done()
       })
     }
+
+    function changeToHapi (path) {
+      var tokens = path.split('/')
+
+      for (var i in tokens) {
```
Comment (geek) [inline web.js]: might be simpler to `tokens.forEach(`

File: web.js (hunk 2 above)
Comment (geek) [inline web.js]: could do `tokens[i].charAt(0) === ':'`

Comment (mirceaalexandru) [inline web.js]: No, protocol is added by server implementation and depends how it is Express/Hapi initialized. 

Path is the for /api/service

**Comment by mirceaalexandru** [comment]
@geek can you take a look please?

**Pull request merged**